components/collapsible-panel/collapsible-panel.js

```diff
@@ -26,7 +27,7 @@ const defaultHeading = 3;
  * @fires d2l-collapsible-panel-expand - Dispatched when the panel is expanded
  * @fires d2l-collapsible-panel-collapse - Dispatched when the panel is collapsed
  */
-class CollapsiblePanel extends FocusMixin(RtlMixin(LitElement)) {
+class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
```

Comment:
Do we need to add skeleton to the `d2l-collapsible-panel-summary-item` as well or just the main component?

Reply:
I think the summary item should support it as well.